### PR TITLE
Ben Kaduk's Discuss

### PR DESCRIPTION
**Pull request by mrdeep1**

### CODE REVIEW

File: draft-ietf-core-new-block.xml [hunk 1]
```diff
@@ -570,21 +570,25 @@
...
-            server MUST use the same token as used for the initial response
-            for returning any Observe triggered responses so that the client
-            can match them up.</t>
+            server MUST use the same token as used for the 2.05
+            (Content) response for returning any Observe triggered responses
+            so that the client can match them up.</t>
 
             <t>The client should then release all of the tokens used for this
-            body unless a resource is being observed.</t>
+            body apart from the one used for tracking observing a resource.</t>
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
            body apart from the one used for tracking an observed resource.</t>
```

File: draft-ietf-core-new-block.xml [hunk 2]
```diff
@@ -875,15 +881,15 @@
...
+        <artwork align="center"><![CDATA[; This defines an array, the elements of which are to be used
 ; in a CBOR Sequence:
 payload = [+ missing-block-number]
 ; A unique block number not received:
 missing-block-number = uint
 ]]></artwork>
       </figure>
 
-      <t></t>
+      <t>This CDDL syntax MUST be followed.</t>
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml:893]: If we go that way, we need to list CDDL as normative.

File: draft-ietf-core-new-block.xml [hunk 3]
```diff
@@ -1059,25 +1067,14 @@ Table 3: Congestion Control Parameters]]></artwork>
...
-        MAX_PAYLOADS values, a peer may continue indicating that there are
-        some missing payloads as all of its MAX_PAYLOADS_SET may not have
-        arrived. How the two peer values for MAX_PAYLOADS are synchronized is
-        out of the scope.</t>
+        potential congestion issues.<list style="empty">
+            <t>Note: Assuming 1500 byte packets, the MAX_PAYLOADS_SET having 10
+            payloads, this is 1500 * 10 * 8 = 120Kbits.  With a maximum delay of
+            2 seconds between MAX_PAYLOADS_SET, this indicates an average
+            minimum speed requirement of 60Kbps for a single body should there
+            be no responses. This is transmission rate is further reduced by
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
            be no responses. This transmission rate is further reduced by
```

File: draft-ietf-core-new-block.xml [hunk 4]
```diff
@@ -1059,25 +1067,14 @@ Table 3: Congestion Control Parameters]]></artwork>
...
-        reconfigured. As a consequence of the two peers having different
-        MAX_PAYLOADS values, a peer may continue indicating that there are
-        some missing payloads as all of its MAX_PAYLOADS_SET may not have
-        arrived. How the two peer values for MAX_PAYLOADS are synchronized is
-        out of the scope.</t>
+        potential congestion issues.<list style="empty">
+            <t>Note: Assuming 1500 byte packets, the MAX_PAYLOADS_SET having 10
+            payloads, this is 1500 * 10 * 8 = 120Kbits.  With a maximum delay of
+            2 seconds between MAX_PAYLOADS_SET, this indicates an average
+            minimum speed requirement of 60Kbps for a single body should there
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
            speed requirement of 60 Kbps for a single body should there
```

File: draft-ietf-core-new-block.xml [hunk 5]
```diff
@@ -1059,25 +1067,14 @@ Table 3: Congestion Control Parameters]]></artwork>
...
-        evaluation. The newly derived value for MAX_PAYLOADS should be used
-        for both ends of this particular CoAP peer link. Note that the CoAP
-        peer will not know about the MAX_PAYLOADS change until it is
-        reconfigured. As a consequence of the two peers having different
-        MAX_PAYLOADS values, a peer may continue indicating that there are
-        some missing payloads as all of its MAX_PAYLOADS_SET may not have
-        arrived. How the two peer values for MAX_PAYLOADS are synchronized is
-        out of the scope.</t>
+        potential congestion issues.<list style="empty">
+            <t>Note: Assuming 1500 byte packets, the MAX_PAYLOADS_SET having 10
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
            <t>Note: Assuming 1500-byte packets and the MAX_PAYLOADS_SET set to 10
```

File: draft-ietf-core-new-block.xml [hunk 6]
```diff
@@ -1059,25 +1067,14 @@ Table 3: Congestion Control Parameters]]></artwork>
...
+            payloads, this is 1500 * 10 * 8 = 120Kbits.  With a maximum delay of
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
            payloads, this corresponds to 1500 * 10 * 8 = 120 Kbits.  With a maximum delay of
```

File: draft-ietf-core-new-block.xml (hunk 4 above)
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
            minimum speed requirement of 60 Kbps for a single body should there
```

File: draft-ietf-core-new-block.xml [hunk 7]
```diff
@@ -1751,6 +1748,14 @@ Payload N: Corresponds to the CoAP message that conveys
       NoSec security mode if either the Q-Block1 or Q-Block2 Options is
       used.</t>
 
+      <t>If OSCORE is not in use, it is also NOT RECOMMENDED to use the
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
      <t>If OSCORE is not used, it is also NOT RECOMMENDED to use the
```

**Review by boucadair — CHANGES_REQUESTED** [review]
I'm just back to work. Thanks for taking care of the edits.

File: draft-ietf-core-new-block.xml (hunk 1 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

File: draft-ietf-core-new-block.xml (hunk 2 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml:893]: Made RFC8610 normative.

File: draft-ietf-core-new-block.xml (hunk 3 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

File: draft-ietf-core-new-block.xml (hunk 4 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

File: draft-ietf-core-new-block.xml (hunk 5 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

File: draft-ietf-core-new-block.xml (hunk 6 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

File: draft-ietf-core-new-block.xml (hunk 4 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Updated

**Pull request merged**